### ISSUE TITLE
Replace Padding with Scale

### ISSUE THREAD

**Issue by ekqt** [ekqt/screenshot#57]
I'm open for suggestions here but I think replacing Padding with a Scale option is a lot better.

- We should integrate scale with a default of 100% and letting the user scale down to 50% and up to 125%. Let's not use steps for this setting.
- We should replace the label of inset padding and inset color with just padding. It should make it a bit more clear on what it does.

**Comment by alefDev-prog** [comment]
Hi, I think this can be solved by adding a width and height attribute to the `image` element in _ClipboardImage.tsx_. The height attribute could be set to auto, so that when the width is changed, the image resizes proportionally.

Since you want to remove the "Padding" setting, do you want to set some default immutable value to the "Padding" instead?

**Comment by alefDev-prog** [comment]
Maybe it's even easier to just use the `transform: scale()` instead of height and with, because we do not need to know the original size of the image.

**Comment by ekqt** [comment]
Do you want to take a crack at it and see? Should I assign it to you? @alefDev-prog

**Comment by alefDev-prog** [comment]
Hi,yes I could try fixing this today

**Comment by ekqt** [comment]
> _I didn't really understand what you meant with not using steps. Do you still want the setting to be in the form of a slider?_

Yes, ideally this should be a slider with min value of 50 and max or 150 with default of 100. The slider should also have a number input with the same parameters. You can take inspo from these sliders.

![image](https://github.com/ekqt/screenshot/assets/77650775/884dac1b-3820-49d0-9697-ebe3b2159ad6)
